src/components/card-section/card-section.tsx

```diff
@@ -44,10 +48,54 @@ const CardSection: React.FunctionComponent<CardSectionProps> = ({
                     );
                 })}
             </Grid>
+        ),
+        [content]
+    );
+    return (
+        <div
+            sx={{
+                gridColumn: ['span 6', null, 'span 8', 'span 12', '4 / span 9'],
+            }}
+        >
+            <div sx={sectionHeadingTopStyles}>
+                <TypographyScale variant="heading5">{title}</TypographyScale>
+                <div
+                    sx={linkWrapperStyles}
+                    onMouseEnter={onLinkEnter}
+                    onMouseLeave={onLinkLeave}
+                >
+                    <Link
+                        href="#"
```

Comment:
I believe we can remove `href` altogether to make this render as a button versus a traditional link.

Reply:
Also I should be able to add `onMouseEnter={onLinkEnter} onMouseLeave={onLinkLeave}` ?

Reply:
Rendering as a button would have SEO impacts though right? 

And coincidentally, the `onMouseEnter` and `onMouseLeave` props are [only used in `Link` if it's rendering as a button](https://github.com/10gen/flora/blob/main/src/Link/Link.tsx#L104), so removing the href would solve having to have the div wrap this.

And yeah if we did that I think we could add it on the link itself, but would have to test it out.